### ISSUE TITLE
Unify Datepicker components

### ISSUE THREAD

**Issue by f1sh1918** [digitalfabrik/entitlementcard#1766]
**Is your feature request related to a problem? Please describe.**
Currently we are using different datepicker components in the application process, birthday extensions and other date extensions.

**Describe the solution you'd like**
- We should try to unify that and use one component for that.
- Maybe we have to use different stylings

**Comment by andrew8er** [comment]
Just for clarification: 
- I found two date related components used in the frontend, `<CustomDatePicker/>` and `<TextField/>` in `DateForm.tsx`.
- `<CustomDatePicker/>` should be used throughout the code base.

Is this correct?

**Comment by f1sh1918** [comment]
> Just for clarification: 
> - I found two date related components used in the frontend, `<CustomDatePicker/>` and `<TextField/>` in `DateForm.tsx`.
> - `<CustomDatePicker/>` should be used throughout the code base.
> 
> Is this correct?

Exactly. I guess we need to support some styling differences for the application form of bavaria
